content/en/docs/tasks/security/authentication/claim-to-header/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Copy Jwt Claim to HTTP Header
+description: Shows how users can copy their jwt claims to http headers.
+weight: 30
+keywords: [security,authentication,jwt,claim]
+aliases:
+    - /docs/tasks/security/istio-auth.html
+    - /docs/tasks/security/authn-policy/
+owner: istio/wg-security-maintainers
+test: yes
+status: Experimental
```

Comment:
Is this feature really experimental? If so, you should also add `{{< boilerplate experimental >}}` at the top of the document text (line 13) as well.